### DISCUSSION TITLE
What's the consequence if we use rm -rf /* in WSL?

### DISCUSSION THREAD

**Discussion by sunmy2019** [microsoft/WSL#6110] [Ideas] [answered]
We know in Liunx, we can delete files while using it. Also in WSL, we can access Windows files. What if we use `rm -rf /*` in WSL or WSL2? Will the Windows files be deleted?

I don't dare to try, but am really curious.

**Comment by Biswa96** [comment, marked as answer]
If the user who has executed System32/wsl.exe or System32/bash.exe has the permission to delete the Windows files and folders **only** then those will be deleted. Which in case is true most of the times.
